app/views/content/landing/career-progression/_collage.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-full-content">
+      <div class="inset">
+        <p class="col-720">
+Teaching is fulfilling career. Not only can you help the pupils you teach, you also get to learn new skills, build your confidence and become your best self. The benefits of being a teacher include:         </p>
+
+        <%= render Content::ChecklistCollageComponent.new(
+          checklist: [
+            "career progression ",
+            "personal development",
+            "teacher's pay"
+          ],
+          image_paths: [
+            "static/content/how-much-do-teachers-get-paid/collage2.jpg",
```

Comment:
Worth considering whether this image (or rather, a new campaign image!) will add anything here - up to you!

Reply:
If yes, to be replaced with campaign image (when available!)

Reply:
@gemmadallmandfe I just left the old image in to work on the content. I'd flagged with Gareth and assumed he'll update it with a new campaign image. But I wasn't sure if the ad team would have a preference to which image is used?